src/com/oltpbenchmark/benchmarks/tpcc/ddls/tpcc-postgres-ddl.sql

```diff
@@ -112,7 +112,7 @@ CREATE TABLE district (
   d_city varchar(20) NOT NULL,
   d_state char(2) NOT NULL,
   d_zip char(9) NOT NULL,
-  PRIMARY KEY (d_w_id,d_id)
+  PRIMARY KEY ((d_w_id,d_id) hash)
```

Comment:
We don't need this since it's only 10 districts per warehouse.

Reply:
done.

Reply:
Not sure I fully understood. If we don't use the hash, then they will on the same tablet.
Either way, this is a small table. So, I'm okay with any of the two options.